src/IotCentral/IotCentral/help/Set-AzIotCentralApp.md

```diff
@@ -1,11 +1,11 @@
-﻿---
-external help file: Microsoft.Azure.PowerShell.Cmdlets.IotCentral.dll-Help.xml
-Module Name: Az.IotCentral
+---
+external help file: Microsoft.Azure.Commands.IotCentral.dll-Help.xml
+Module Name: AzureRM.IotCentral
```

Comment:
@cle5eland it looks like `Az` was changed to `AzureRM` in this file (and the `dll.Help.xml` file above changed). Please make sure that you don't have `AzureRM.IotCentral` installed on your machine when you generate the markdown files -- only the local `Az.IotCentral` module that you build should be imported. Let me know if you have any questions with this.

Reply:
Makes sense, let me fix.

Reply:
@cormacpayne should be fixed now